addons/common/functions/fnc_setupLocalUnitsHandler.sqf

```diff
@@ -15,6 +15,8 @@
  *
  * Public: No
  */
+// Ignore UAV/Drone AI
+#define IGNORE_BASE_UAVPILOTS "B_UAV_AI", "O_UAV_AI", "UAV_AI_base_F"
```

Comment:
This is not a translation? Why is this in here?

Reply:
cuz i updated my master

Reply:
How did you do that? If you do it correctly it doesn't push other stuff into your pull request